tests/integration/views/account/test_password_change.py

```diff
@@ -31,11 +31,23 @@ def tearDown(self):
 
     @requests_mock.mock()
     @patch('frontstage.controllers.party_controller.get_respondent_party_by_id')
-    def test_account_password_change_success(self, mock_request, get_respondent_party_by_id):
+    @patch('frontstage.controllers.party_controller.get_survey_list_details_for_party')
```

Comment:
why do we need survey_list for changing password?

Reply:
Have you ran the test?  It raises a warning because of urls that aren't mocked.  The reason is because after you change a password, you're redirected to the todo page.
The test makes sure you're on the todo page, but the test warns you that you can't get all the info for it.  Which is kind of surprising, normally it outright fails if a mock hasn't been done, possibly because of the way it uses threads to build its own cache? 

Either way, this doesn't change functionality, it just suppresses the error by patching what get_survey_list_details_for_party (the function that does all the heavy lifting for the todo page) would give you